webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx

```diff
@@ -136,6 +142,8 @@ const ExperimentSingleTrialTabs: React.FC<Props> = ({
       const response = await getTrialDetails({ id: trialId }, { signal: canceler.signal });
       onTrialUpdate?.(response);
       setTrialDetails(response);
+      const remainingDays = await getTrialRemainingLogRetentionDays({ id: trialId });
```

Comment:
if the order of `getTrialDetails` and `getTrialRemainingLogRetentionDays` is not important, use `Promise.all` or `Promise. allSettled`

Reply:
done!